shell/platform/embedder/embedder_layers.cc

```diff
@@ -35,6 +38,33 @@ void EmbedderLayers::PushBackingStoreLayer(const FlutterBackingStore* store) {
   layer.size.width = transformed_layer_bounds.width();
   layer.size.height = transformed_layer_bounds.height();
 
+  auto paint_region_rects = std::make_unique<std::vector<FlutterRect>>();
+  paint_region_rects->reserve(paint_region_vec.size());
+
+  for (const auto& rect : paint_region_vec) {
+    auto transformed_rect =
+        root_surface_transformation_.mapRect(SkRect::Make(rect));
+    paint_region_rects->push_back(FlutterRect{
```

Comment:
I _think_ this could be `emplace_back(transformed_rect.x(), ...)`?

Reply:
Flutter rect doesn't have a 4 argument constructor so the compiler is not too happy about it.